activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -78,6 +78,18 @@ def #{method_name}=(value)           # def includes_values=(value)
 
     alias extensions extending_values
 
+    def load_associations(arr, *args)
```

Comment:
Can we call the `*args` of `associations`?

Reply:
Do you mean a variable rename here or something else?

Reply:
It's tough to understand what `args` mean here, yes. As a reader I'm confused by `args.each do |associations|`. So it holds an array of arrays of associations? Or something else?